tinygrad/shape/symbolic.py

```diff
@@ -63,8 +62,10 @@ def __rmul__(self, b:int): return self*b
 
   # *** complex ops ***
 
-  def __floordiv__(self, b:int, factoring_allowed=True):
+  def __rfloordiv__(self, b:int): raise RuntimeError(f"not supported: {b} // {self}")
+  def __floordiv__(self, b:Union[Node,int], factoring_allowed=True):
     assert b != 0
+    if isinstance(b, Node) and b > self >= 0: return NumNode(0)
```

Comment:
`b > self >= 0` if b is Node? What is this?

Reply:
This was a simplification case, I made this part explicit for what we support, so no dangling `ModNode` or `DivNode` with `b: Node` is allowed now. Only support if it can be simplified.

The old comparison has false positive case because the bool of a `LtNode` is false only if both min and max are 0, I have updated the code and add a test case.